.env

```diff
@@ -0,0 +1,3 @@
+GLOBAL_CLIENT_ID=pSm3810KgGx5qnjckjRGoWuBLlU0Hjgl
+GLOBAL_CLIENT_SECRET=OG1G34icGnYQKjtV26Q_qokQe6bafT8K3ecdHwIQ4pcCEiKjfQNBpZkzJdVxg_8s
+DOMAIN=happyr.eu.auth0.com
```

Comment:
I must have made a mistake. Thank you. 

(I have a hard time running tests locally)